src/widgets/current-refined-values/defaultTemplates.js

```diff
@@ -1,15 +1,29 @@
 export default {
   header: '',
-  item:
-    '' +
-    '{{#label}}' +
-    '{{label}}' +
-    '{{^operator}}:{{/operator}}' +
-    ' ' +
-    '{{/label}}' +
-    '{{#operator}}{{{displayOperator}}} {{/operator}}' +
-    '{{#exclude}}-{{/exclude}}' +
-    '{{name}} <span class="{{cssClasses.count}}">{{count}}</span>',
+  item: itemTemplate,
   clearAll: 'Clear all',
   footer: '',
 };
+
+function itemTemplate({
+  label,
+  operator,
+  displayOperator,
+  exclude,
+  name,
+  count,
+  cssClasses,
+  query,
+}) {
+  const computedOperator = operator ? displayOperator : '';
+  const computedLabel = label
+    ? `${label} ${computedOperator || ':'} `
+    : computedOperator;
+  const computedCount = query
+    ? ''
+    : `<span class="${cssClasses.count}">${count === undefined
+        ? 0
+        : count}</span>`;
```

Comment:
This should not eslint pass since we lint for `no-nested-ternary`

Reply:
It seems it didn't raise any error on linting, but I'll replace it 👍